addons/mail_plugin/models/__init__.py

```diff
@@ -2,4 +2,5 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from . import ir_http
+from . import res_partner_iap
```

Comment:
Technically, res_partner_iap should come after res_partner due to alphabetical order